.ci.yaml

```diff
@@ -4692,13 +4690,12 @@ targets:
   - name: Windows customer_testing
     enabled_branches:
       - master
-    recipe: flutter/flutter
+    recipe: flutter/flutter_drone
     timeout: 60
     properties:
-      validation: customer_testing
-      validation_name: Customer testing
+      shard: customer_testing
       tags: >
-        ["framework", "hostonly", "windows"]
+        ["framework", "hostonly", "shard", "windows"]
```

Comment:
so, we're expecting presubmits to not reflect this change, because compiling ci.yaml to lucicfg configs happens post-submit, right?

Reply:
That is correct, I've been running them manually with led https://luci-milo.appspot.com/raw/build/logs.chromium.org/flutter/led/godofredoc_google.com/c8307d6b5bb5f03b08393c1a5e1aaedc9ef2c2064b14521627832c38c09cc26a/+/build.proto

Reply:
This is the run for mac: https://chromium-swarm.appspot.com/task?id=6607d9ba068fc310

Reply:
And this one for win: https://chromium-swarm.appspot.com/task?id=6607dbd699281310

Reply:
looks like win is failing:

```
║ ProcessException: The system cannot find the file specified.
║ 
║   Command: ci.bat 
║ #0      _ProcessImpl._start (dart:io-patch/process_patch.dart:402:33)
║ #1      Process.start (dart:io-patch/process_patch.dart:38:20)
║ #2      startCommand (file:///C:/b/s/w/ir/x/w/flutter/dev/bots/run_command.dart:97:47)
║ #3      runCommand (file:///C:/b/s/w/ir/x/w/flutter/dev/bots/run_command.dart:165:33)
║ #4      _runCustomerTesting (file:///C:/b/s/w/ir/x/w/flutter/dev/bots/test.dart:1583:9)
║ <asynchronous suspension>
║ #5      _runFromList (file:///C:/b/s/w/ir/x/w/flutter/dev/bots/test.dart:2175:5)
║ <asynchronous suspension>
║ #6      main (file:///C:/b/s/w/ir/x/w/flutter/dev/bots/test.dart:247:5)
║ <asynchronous suspension>
║ 
║ The test.dart script should be corrected to catch this error and call foundError().
║ Some tests are likely to have been skipped.
```

Reply:
Taking a look, I realized I didn't run on win before.

Reply:
The problem was that windows wants bat files to be executed using full path rather than a relative one.

Reply:
lol